bibliopixel/led.py

```diff
@@ -18,9 +18,9 @@ def __init__(self, driver):
         self._data = []
         self._driver = driver
 
-    def setData(self, data):
+    def setData(self, colors, pos):
         self._reading.wait()
-        self._data = data
+        self._data = (colors, pos)
```

Comment:
Oh... because using multi driver, each driver may get a subset of the full list. That makes sense. I assume that's all threadsafe?